doc/Makefile.am

```diff
@@ -54,56 +54,7 @@ portal_files = 								\
 	$(top_srcdir)/data/org.freedesktop.impl.portal.Wallpaper.xml		\
 	$(NULL)
 
-xml_files = 								\
-	portal-org.freedesktop.portal.Account.xml 			\
-	portal-org.freedesktop.portal.Background.xml			\
-	portal-org.freedesktop.portal.Camera.xml			\
-	portal-org.freedesktop.portal.Device.xml			\
-	portal-org.freedesktop.portal.Documents.xml			\
-	portal-org.freedesktop.portal.Email.xml 			\
-	portal-org.freedesktop.portal.FileChooser.xml 			\
-	portal-org.freedesktop.portal.FileTransfer.xml			\
-	portal-org.freedesktop.portal.Flatpak.UpdateMonitor.xml 			\
```

Comment:
> Note that this removes the
> FlatpakUpdateMonitor from the explicitly mentioned files but
> gdbus-codegen seems to do the right thing with this now.

Was this true as long ago as GLib 2.60?

Reply:
I tried this in a Fedora 30 podman which uses 2.60.7 and the answer is "no, apparently not". However: the `UpdateMonitor` file was only used in `doc/Makefile.am`, `git grep` shows nothing else referencing it. So the only drawback here appears to be that the documentation can't be built on old versions? And that's only because of the `<xi:include>`, without that it'd happily complete.

Reply:
re-visited this and: `UpdateMonitor` wasn't used for generation of files (the code always relied on this to be generated) but for listing the results of the gdbus-codegen files, to be used e.g in `CLEANFILES`. Updated the patchset accordingly.